dom.js

```diff
@@ -19,3 +19,176 @@ function listFunc() {
+  },
+
+  {
+    id: 4,
+    title: 'Profesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    prog1: 'css',
+    prog2: 'html',
+    prog3: 'Ruby',
+    btn1: 'See Project',
+  },
+  {
+    id: 5,
+    title: 'Profesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    prog1: 'css',
+    prog2: 'html',
+    prog3: 'Ruby',
+    btn1: 'See Project',
+  },
+
+  {
+    id: 6,
+    title: 'Profesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    prog1: 'css',
+    prog2: 'html',
+    prog3: 'Ruby',
+    btn1: 'See Project',
+  },
+  {
+    id: 7,
+    title: 'Profesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    prog1: 'css',
+    prog2: 'html',
+    prog3: 'Ruby',
+    btn1: 'See Project',
+  },
+];
+
+function cards() {
+  sect3.forEach((cardsec) => {
+    projcard4.innerHTML += `
+    <div class="card3">
+    <div class="chips">
+       <h3>Professional Art<br>${cardsec.title}</h3>
+       <p>${cardsec.description}</p>
+       <ul class="card-ul">
+           <li>${cardsec.prog1}</li>
+           <li>${cardsec.prog2}</li>
+           <li>${cardsec.prog3}</li>
+       </ul>
+      </div>
+      <button type="button" class="cardbtn" onclick="activePop()">${cardsec.btn1}</button> 
+  </div>`;
+  });
+}
+window.addEventListener('load', cards);
+
+// Popup Starts Here
+const pop = document.querySelector('.popup');
+const overlay = document.querySelector('.overlay');
+const popwin = {
+  id: 1,
+  title: 'Keeping track of hundreds of components website',
+  image: 'popdesk.png',
+  description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsumhas been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  prog1: 'css',
+  prog2: 'html',
+  prog3: 'bootstrap',
+  prog4: 'Ruby on rails',
+  btn1: 'see live',
+  btn2: 'see source',
+};
```

Comment:
- [ ] Although you put in a great effort in creating the popup :+1:, please keep in mind that it is a project requirement for the popup to be populated with the information of the respective work card. I have noticed that you have used the same popup for all work cards, and it is not changing based on the specific card that is being clicked.
To meet the project requirements, you should modify the popup so that it displays the information specific to the work card that is being clicked. This will enhance the user experience and improve the overall functionality of the project.